lib/ChangeDueDateDialog/ChangeDueDateDialog.js

```diff
@@ -264,17 +273,23 @@ class ChangeDueDateDialog extends React.Component {
         open={open}
         label={modalLabel}
       >
-        <BodyComponent
-          user={user}
-          alerts={alerts}
-          stripes={stripes}
-          onDueDateChanged={this.handleDueDateChanged}
-          onDueDateChangeFailed={this.handleDueDateChangeFailed}
-          onCancel={this.handleCancel}
-          loans={this.loans()}
-          requestCounts={this.getOpenRequestCounts()}
-          dueDatesChanged={dueDatesChanged}
-        />
+        {
+          isEmpty(loans) ?
+            <div className={css.spinner}>
```

Comment:
Could you please give a try for the Layout component instead of the auxiliary div?

```
        <Layout className="textCentered">
          <Spinner />
        </Layout>
```